test/plugins/cleanupIds.26.svg.txt

```diff
@@ -0,0 +1,29 @@
+Should handle non-ASCII IDs and resolve not correctly URI encoded references.
+
+See: https://github.com/svg/svgo/issues/1981
+
+===
+
+<svg xmlns="http://www.w3.org/2000/svg">
+    <defs>
+        <linearGradient id="渐变_1" x1="0%" y1="0%" x2="100%" y2="0%">
+			      <stop offset="10%" stop-color="#c71700" />
+            <stop offset="60%" stop-color="orange" />
+            <stop offset="100%" stop-color="#5a2100" />
+        </linearGradient>
+    </defs>
+    <rect x="30" y="30" height="150" width="370" fill="url(#渐变_1)" />
+</svg>
+
+@@@
+
+<svg xmlns="http://www.w3.org/2000/svg">
+    <defs>
+        <linearGradient id="a" x1="0%" y1="0%" x2="100%" y2="0%">
+            <stop offset="10%" stop-color="#c71700"/>
+            <stop offset="60%" stop-color="orange"/>
```

Comment:
```suggestion
            <stop offset="100%" stop-color="#5a2100" />
        </linearGradient>
    </defs>
    <rect x="30" y="30" height="150" width="370" fill="url(#渐变_1)" />
</svg>

@@@

<svg xmlns="http://www.w3.org/2000/svg">
    <defs>
        <linearGradient id="a" x1="0%" y1="0%" x2="100%" y2="0%">
            <stop offset="10%" stop-color="#c71700"/>
```

Just to keep the tests minimal, can we omit the two lines with offset 60%? We prefer for the tests to be as small as possible, so they just focus on what they're intended to test.